lua/telescope/_extensions/file_browser/actions.lua

```diff
@@ -417,6 +417,20 @@ fb_actions.goto_cwd = function(prompt_bufnr)
   current_picker:refresh(finder, { reset_prompt = true, multi = current_picker._multi })
 end
 
+--- Change working directory of nvim to the path of |builtin.file_browser|.
+---@param prompt_bufnr number: The prompt bufnr
+fb_actions.change_cwd = function(prompt_bufnr)
+  local current_picker = action_state.get_current_picker(prompt_bufnr)
+  local finder = current_picker.finder
+  vim.cmd("cd " .. finder.path)
+  finder.files = true
```

Comment:
As I understand, this is supposed to be triggered in `file_browser` mode. Why do we need L426-431, inclusive, then?

Reply:
This was in order to update the picker title, but there are things I still want to change ...